src/screens/CandidateProfile/index.js

```diff
@@ -134,7 +134,7 @@ export default function CandidateProfile({ route }) {
                       }}
                       onPress={() =>
                         navigation.navigate("ThemeByCandidate", {
-                          idCandidat: IDCandidat,
+                          idCandidat: IDCandidate,
```

Comment:
We should rename the key ? To make uniform "idcandidate"

Reply:
Yes I think also, but it needs to change it everywhere on the app, and I'm running out of time for now...